src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,42 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.UserNotAllowed;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMAL_LOGIN_LENGTH = 6;
+    private static final int MINIMAL_USER_AGE = 18;
+    private static final int MINIMAL_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
```

Comment:
It's better to decide whether to use `@BeforeEach` or `@BeforeAll` for initializing your `RegistrationServiceImpl` class instance. If you need a fresh instance before each test, use `@BeforeEach`. If the same instance can be reused across tests, use `@BeforeAll`. However, since `RegistrationServiceImpl` does not maintain any state that changes between tests, you could use `@BeforeAll` to initialize it once for all tests.